internal/services/network/express_route_circuit_resource.go

```diff
@@ -279,10 +279,6 @@ func resourceExpressRouteCircuitUpdate(d *pluginsdk.ResourceData, meta interface
 		return fmt.Errorf("retrieving %s : %s", id, err)
 	}
 
-	if err := client.CreateOrUpdateThenPoll(ctx, *id, *existing.Model); err != nil {
-		return fmt.Errorf("updating %s : %+v", id, err)
-	}
-
```

Comment:
should we be removing this here? the comment on line 272 suggests this is necessary

Reply:
The changes have been reverted